src/wallet/walletdb.cpp

```diff
@@ -665,7 +665,8 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
         } else if (strType != DBKeys::BESTBLOCK && strType != DBKeys::BESTBLOCK_NOMERKLE &&
                    strType != DBKeys::MINVERSION && strType != DBKeys::ACENTRY &&
                    strType != DBKeys::VERSION && strType != DBKeys::SETTINGS &&
-                   strType != DBKeys::PRIVATESEND_SALT && strType != DBKeys::COINJOIN_SALT) {
+                   strType != DBKeys::PRIVATESEND_SALT && strType != DBKeys::COINJOIN_SALT &&
+                   strType != DBKeys::FLAGS) {
```

Comment:
21127 should drop this: https://github.com/dashpay/dash/pull/6017/commits/e3968faeb3e96b6b4cf15b0f9aef5d92d3f728f6#diff-49a07da77d79017222ba989b99048c0a629d3230988ce7b6b74df0d14f727626R557-R563 no?

Reply:
good catch, fixed